railties/lib/rails/application/routes_reloader.rb

```diff
@@ -4,11 +4,13 @@ module Rails
   class Application
     class RoutesReloader
       attr_reader :route_sets, :paths
-      delegate :execute_if_updated, :execute, :updated?, to: :updater
+      attr_accessor :eager_load
+      delegate :execute_if_updated, :updated?, to: :updater
```

Comment:
I think `execute_if_updated` should also be defined here to run the eager load code.